lib/game.dart

```diff
@@ -10,12 +10,31 @@ import 'package:ordered_set/ordered_set.dart';
 
 import 'components/component.dart';
 import 'position.dart';
+import 'components/events/gestures.dart';
+import 'flame.dart';
 
 /// Represents a generic game.
 ///
 /// Subclass this to implement the [update] and [render] methods.
 /// Flame will deal with calling these methods properly when the game's widget is rendered.
 abstract class Game {
+  TapGestureRecognizer _createTapGestureRecognizer() => TapGestureRecognizer()
+    ..onTapUp = (TapUpDetails details) {
```

Comment:
inline function

Reply:
I can't use inline functions here because onTapUp, onTapDown and onTapCancel are void and the compiler complains.

Reply:
but GestureTapUpCallback is also void... I'm suggesting
`..onTapUp = (TapUpDetails details) => onTapUp(details)`
that doesn't work? if not, you can mark all three of theses comments as resolved.

Reply:
yeah, it breaks the compilers, I THINK this actually happens because the compiler get lost because we are using the `..` operator.